src/js/components/modals/ServiceScaleFormModal.js

```diff
@@ -106,6 +106,17 @@ class ServiceScaleFormModal extends mixin(StoreMixin) {
           id: service.id,
           scaleBy: parseInt(instances, 10)
         });
+      } else if (service instanceof Pod) {
+        let spec = Object.assign({}, service.getSpec().get(),
```

Comment:
Let's have one util that deals with all kinds of services. I also think that we could add this to the `ServiceUtil` instead of introducing yet another util collection.